plugins/http/Cargo.toml

```diff
@@ -13,6 +13,13 @@ links = "tauri-plugin-http"
 rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 
+[package.metadata.platforms.support]
+windows = { level = "full", notes = "" }
+linux = { level = "full", notes = "" }
+macos = { level = "full", notes = "" }
+android = { level = "none", notes = "" }
+ios = { level = "none", notes = "" }
```

Comment:
@lucasfernog can you clarify this one? May be working on mobile without swift/kotlin plugins

Reply:
doing a bit of testing for this pr rn and at least in the example the `fetch()` call never resolves.